setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='tqdm',
-    version='1.0',
+    version='1.1',
```

Comment:
I think it is better to change version number in a separate commit, with a git tag and release notes.